### PR TITLE
fix: search bugs

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Fixes https://github.com/snapshot-labs/snapshot/issues/3641
Fixes https://github.com/snapshot-labs/snapshot/issues/3653
Fixes https://github.com/snapshot-labs/snapshot/issues/3652

### CODE REVIEW

**Comment by samuveth** [comment]
@j0hnfl0w can you resolve the conflicts

File: src/components/SpaceProposalVotesModal.vue [hunk 1]
```diff
@@ -33,13 +33,14 @@ const {
...
+  isResolvingEns,
+  searchAddress
 } = useProposalVotes(props.proposal, 20, props.userVote, votesQuery);
 
 useIntersectionObserver(
   votesEndEl,
-  ([{ isIntersecting }], observerElement) => {
-    if (props.open && isIntersecting) {
+  ([{ isIntersecting }]) => {
+    if (props.open && isIntersecting && searchAddress.value === '') {
```
[the remark this answers is not in the document]
Comment (j0hnfl0w) [reply src/components/SpaceProposalVotesModal.vue:41]: cant reproduce double call with address call and for ENS too
https://www.loom.com/share/e5a99c8197fb4afc83c5235bfd565a05

**Comment by samuveth** [comment]
@j0hnfl0w can you make it so that if the user connects the wallet after the proposal page is loaded it will load the users vote and render it first in the list?

**Comment by samuveth** [comment]
Great work

**Pull request merged**